Neos.Flow/Classes/Reflection/ReflectionService.php

```diff
@@ -265,8 +265,9 @@ public function getAllClassNames(): array
         if (!$this->initialized) {
             $this->initialize();
         }
-
-        return array_keys($this->classReflectionData);
+        $classNames = array_keys($this->classReflectionData);
+        sort($classNames);
```

Comment:
as karsten wrote

> Why the sorting? Is that the central replacement for the removed sort calls later on?

yes as the structure is not internally sorted anymore but `getAllClassNames` guaranteed A-Z order before i thought we should continue to do so.
On the other hand, `getAllClassNames` can be considered rather low level and is only used once so we can also put sorting on the callers site - maybe no sorting is needed for every usecase.